kloppy/_providers/impect.py

```diff
@@ -0,0 +1,43 @@
+from kloppy.config import get_config
+from kloppy.domain.models.impect.event import ImpectEventFactory
+from kloppy.infra.serializers.event.impect import (
+    ImpectDeserializer,
+    ImpectInputs,
+)
+from kloppy.domain import EventDataset, Optional, List, EventFactory
+from kloppy.io import open_as_file, FileLike
+
+
+def load(
+    event_data: FileLike,
+    lineup_data: FileLike,
+    event_types: Optional[List[str]] = None,
+    coordinates: Optional[str] = None,
+    event_factory: Optional[EventFactory] = None,
+) -> EventDataset:
+    """
+    Load Impect event data into a [`EventDataset`][kloppy.domain.models.event.EventDataset]
+
+    Parameters:
+        event_data: filename of json containing the events
+        lineup_data: filename of json containing the lineup information
+        event_types:
+        coordinates:
+        event_factory:
+    """
+    deserializer = ImpectDeserializer(
+        event_types=event_types,
+        coordinate_system=coordinates,
+        event_factory=event_factory
+        or get_config("event_factory")
+        or ImpectEventFactory(),
```

Comment:
This `ImpectEventFactory()` is copy-pasted from the StatsBomb example but it is doesn't have any function. I think it can also be removed from the StatsBomb code by the way.